src/test/java/com/salesforce/dynamodbv2/UpdateTest.java

```diff
@@ -233,26 +237,30 @@ private void runUpdateConditionalOnGsiRkTest(TestArgument testArgument, boolean
                 .withKey(ItemBuilder.builder(testArgument.getHashKeyAttrType(), HASH_KEY_VALUE)
                     .rangeKey(S, RANGE_KEY_OTHER_S_VALUE).build())
                 .addExpressionAttributeValuesEntry(
-                    ":newValue", createStringAttribute(INDEX_FIELD_VALUE + TABLE3 + org + "Updated"))
+                    ":hkValue", createAttributeValue(S, GSI2_HK_FIELD_VALUE + TABLE3 + org + "Updated"))
```

Comment:
Tweaked a couple of test cases so that we don't modify just one but not all fields of a secondary index, without changing what the test case is essentially testing. And then disabled one test case for hash partitioning, where the scenario being tested requires not updating all secondary index fields at once.